### PR TITLE
Add sorting functionality to table

### PR DESCRIPTION
**Pull request by helrond**
Uses an existing Javascript to implement sorting functionality on table.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,20 +7,22 @@
...
   <title>space barnacle</title>
   <!-- CDNs should be used for production deployment -->
-  <!-- <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/css/bootstrap.min.css" integrity="sha384-rwoIResjU2yc3z8GV/NPeZWAv56rSmLldC3R/AZzGRnGxQQKnKkoFVhFQhNUwEyJ" crossorigin="anonymous"> -->
+  <!-- <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.15/css/dataTables.bootstrap.min.css"> -->
+  <!-- <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous"> -->
   <!-- local copy of CSS for development -->
-  <link href="vendor/css/bootstrap.min.css" rel="stylesheet">
+  <link href="vendor/DataTables/datatables.min.css" rel="stylesheet">
+  <link href="vendor/bootstrap/bootstrap.min.css" rel="stylesheet">
 </head>
```
Comment (EcZX) [inline index.html]: We should add "I live in misery and regret".

File: index.html [hunk 2]
```diff
@@ -33,8 +35,8 @@ <h1 class="pull-left">space barnacle <small>thinking inside the box</small></h1>
               <div class="modal-dialog">
                 <div class="modal-content">
                   <div class="modal-header">
-                    <h2 class="modal-title">Help</h2>
                     <button type="button" class="close" data-dismiss="modal">&times;</button>
+                    <h2 class="modal-title text-left">Help</h2>
                   </div>
                   <div class="modal-body text-left">
                     <p>space barnacle helps you find a box you can fit into.</p>
```
Comment (eswanson611) [inline index.html]: Space Barnacle description is gold. #leaving in a box

**Comment by jwelbs** [comment]
I like how the Captain's Log has a "beam me up" button before the results display. Does our table sort in "real time" or do you push a button and get a list of boxes?

File: js/app.js [hunk 3]
```diff
@@ -19,3 +19,7 @@ function displayData() {
 }
 
 displayData();
+
+$(document).ready(function() {
```
Comment (jwelbs) [inline js/app.js]: Looks good.

**Comment by helrond** [comment]
@jwelbs The table column headings allow for sorting. You'll be able to see how that works when I demo the application!

**Pull request merged**